src/components/ContentWidgets/Separator/SeparatorEditor.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { WidgetEditorProps } from "../../ContentMapping/ContentMapping";
+import { Separator } from "./Separator";
+
+/**
+ * SeparatorEditor edits the width and color of a separator widget
+ *
+ * Last Modified
+ * Jennifer Tao
+ * July 10, 2019
+ */
+export const SeparatorEditor: React.FC<WidgetEditorProps> = ({
+  editedContent,
+  setEditedContentOnChange
+}) => {
+  const defaultColor = { r: 66, g: 13, b: 171 };
```

Comment:
Declare this const in a constants file instead, otherwise you will create a new version of this when there are multiple editors for a Separator. Small optimizations.